webextensions/manifest/optional_permissions.json

```diff
@@ -125,7 +125,7 @@
           "__compat": {
             "support": {
               "chrome": {
-                "version_added": true
+                "version_added": "≤81"
```

Comment:
@queengooborg This and several other permissions weren't identified as supported in Chrome until Chrome optional permissions update [#22034](https://github.com/mdn/browser-compat-data/pull/22034) which equates to version 121.

Reply:
Crap...121 is way too recent to create a range.  Think we can track down the exact version numbers, or at least a version two or more years old?

Reply:
This was the commit adding support for optional permissions: https://chromium.googlesource.com/chromium/src/+/d9c1ade4d29daf457cadbfdf15c71e92e5ffeebb